spec/controllers/api/admin/chapters/chapter_notes_controller_spec.rb

```diff
@@ -30,9 +30,9 @@
       let(:chapter) { create :chapter }
 
       it 'returns not found if record was not found' do
-        get :show, params: { chapter_id: chapter.to_param }, format: :json
-
-        expect(response.status).to eq 404
```

Comment:
I think the advantage of texting the 404 status,
is that it's closer to the production behaviour.

We test the behaviour we expect in production.